src/deliver/render_message.rs

```diff
@@ -177,6 +199,13 @@ fn truncate(s: &str, max_chars: usize) -> String {
     result.trim().to_string()
 }
 
+fn create_links(urls: &str)->  String {
```

Comment:
this function is not needed. let's format the string directly in the `create_link` helper on the line 37